### PR TITLE
Update Uzbekistan holidays, add l10n support

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Update Uzbekistan holidays, add l10n support (en_US, uk, **uz**).

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've run `make pre-commit`, it didn't generate any changes
- [x] I've run `make test`, all tests passed locally

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/uzbekistan.py [hunk 1]
```diff
@@ -9,51 +9,87 @@
...
+        - `Labor Code 21.12.1995 <https://lex.uz/docs/-142859>`_
+        - `Labor Code 28.10.2022 <https://lex.uz/docs/-6257288>`_
     """
 
     country = "UZ"
+    default_language = "uz"
+    # Estimated label.
+    estimated_label = tr("%s* (*taxminiy)")
+    # %s (Observed).
+    observed_label = tr("%s (ko‘chirilgan)")
```
Comment (arkid15r) [inline holidays/countries/uzbekistan.py:34]: Shouldn't we use ' instead of ‘ here?

File: holidays/countries/uzbekistan.py [hunk 2]
```diff
@@ -9,51 +9,87 @@
...
+    References:
+        - https://en.wikipedia.org/wiki/Public_holidays_in_Uzbekistan
+        - `Labor Code 21.12.1995 <https://lex.uz/docs/-142859>`_
+        - `Labor Code 28.10.2022 <https://lex.uz/docs/-6257288>`_
     """
 
     country = "UZ"
+    default_language = "uz"
+    # Estimated label.
+    estimated_label = tr("%s* (*taxminiy)")
```
Comment (arkid15r) [inline holidays/countries/uzbekistan.py:32]: * need to be removed to align with the recently submitted changes

**Review by arkid15r — COMMENTED** [review]
Looks good!

File: holidays/countries/uzbekistan.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/uzbekistan.py:34]: According to [Wiki](https://en.wikipedia.org/wiki/Uzbek_alphabet#Distinct_characters), the most correct symbol to use is `ʻ` (U+02BB), but official sites (including https://lex.uz) use `‘` (U+2018). That's why I used U+2018 as well.

Comment (arkid15r) [reply holidays/countries/uzbekistan.py:34]: You're right, this is the correct one.

**Pull request merged**